### ISSUE TITLE
Converting GQLDocument to String: multiple interfaces are not separated by ampersands

### ISSUE THREAD

**Issue by andrey-nakin** [apollographql/apollo-kotlin#4149]
I have  a GraphQL schema with a type implementing two interfaces, like this:

```
interface IA1 {
  id: ID!
}

interface IA2 {
  fieldA: String
}

type TypeA implements IA1 & IA2 {
  id: ID!
  fieldA: String
}
```

I can successfully parse such a schema into a `GQLDocument` using `apollo-ast` library. But when I do an opposite translation, from `GQLDocument` to `String`, the ampersand symbol between the interfaces disappears and the resulting type definition looks like this:

```
type TypeA implements IA1 IA2 {
  id: ID!
  fieldA: String
}
```

Here is my Java code that converts `GQLDocument` to a string:

```
    String toString(GQLDocument src) {
        return GqlnodeKt.toUtf8(src, "  ");
    }
```

**Comment by andrey-nakin** [comment]
> Is this causing trouble with other parsers?
Yes, particularly [Tyk](https://github.com/TykTechnologies/tyk) fails to load such a schema.

> do you want to try a PR?
Sorry, I'm not a Kotlin developer.

**Comment by BoD** [comment]
> Yes, particularly Tyk fails to load such a schema.

Good to know!

No worries about the PR, I'll have a look 😊 Thanks for reporting this.

**Comment by BoD** [comment]
Fix available in v3.3.1.
